pkg/proxy/snapshot.go

```diff
@@ -66,7 +66,7 @@ func (ops V2DataEngineProxyOps) VolumeSnapshot(ctx context.Context, req *rpc.Eng
 		snapshotName = util.UUID()
 	}
 
-	_, err = c.EngineSnapshotCreate(req.ProxyEngineRequest.EngineName, snapshotName)
+	_, err = c.EngineSnapshotCreate(req.ProxyEngineRequest.EngineName, snapshotName, nil)
```

Comment:
Because longhorn-spark-engine has no branch yet before GA, the interface incompatibility is fine. However, we need to consider exported interface compatibility in the future after GA.

cc @DamiaSan @shuo-wu